lib/js/components/Buttons/Button/Button.vue

```diff
@@ -41,7 +41,7 @@
 		<wnl-icon
 			v-if="state === STATES.LOADING"
 			class="a-button__loadingIcon"
-			:icon="ICONS.FA_SPINNER_THIRD"
```

Comment:
should we also migrate the old icon in `Tile.vue`? If we do that we could remove this icon from production build

Reply:
Sure.

Reply:
can we remove `FA_SPINNER_THIRD` now ?

Reply:
Yes, I have this change but I'm waiting for DR finish to give this PR to review again.

Reply:
There are more icons that are not used in DS/Platform.

Reply:
> Yes, I have this change but I'm waiting for DR finish to give this PR to review again.
I still don't understand what you cannot remove it now.

> There are more icons that are not used in DS/Platform. F.e. FA_FILE_PEN_SOLID FA_BOOK_SPARKLES_SOLID
I'm not sure I understood intention of this comment. Can you elaborate?

Reply:
Icon removed, discussed.